src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/CuratedMainPagePageObject.java

```diff
@@ -31,6 +32,25 @@
   private WebElement trendingVideos;
   @FindBy(css = ".mobile-prefooter")
   private WebElement mobilePrefooter;
+  @FindBys(@FindBy(css = "div.curated-content a"))
+  private List<WebElement> curatedContentItems;
```

Comment:
this selector is already defined here: https://github.com/Wikia/selenium-tests/pull/745/files#diff-e460dfeac19aff73ac4e286d2671564fR21

Those two classes are couple together, maybe you can extend CuratedPageObject here instead of MercuryBasePageObject?